pages/api-reference/funding-sources/create-funding-source-for-customer.mdx

```diff
@@ -133,6 +133,8 @@ dwolla
 
 ### Request and response - using on-demand authorization
 
+For an example of using on-demand authorization, see the example for adding a customer funding source using Plaid Link [here](https://github.com/Dwolla/integration-examples).
```

Comment:
Should we include a full link to the plaid example like what you did in the API reference? https://github.com/Dwolla/integration-examples/tree/main/packages/plaid-funding-source